packages/contracts-bedrock/deploy/001-InitImplementations.deploy.ts

```diff
@@ -7,7 +7,7 @@ import '@eth-optimism/hardhat-deploy-config'
 
 const upgradeABIs = {
   L2OutputOracleProxy: async (deployConfig) => [
-    'initialize(bytes32,uint256,address,address)',
```

Comment:
Is this a related change / fix? Seems worth a comment in PR description or a separate PR. And is it required for CI?

Reply:
I would like to write a unit test for this. I was waiting on doing a full deploy to make sure that this fixed the issue. This specific line was blocking a full deploy - we updated the ABI but did not change this so hardhat would choke on deployments.

Reply:
@mslipper when did this get fixed on develop? This is pretty up to date & the PR that broke it is about a week old

Reply:
Oh, I found the old PR. This commit is dropped.